NSS-Day-Cohort-49/react-nutshell-state-slingers
# HCR-ArticlesList

#### Changes Made
1. Added .jpg profile picture.
1. Added Article files to create Article List, Article Provider, Article Form, Article Card and Article css.
1. Articles list displays and form posts and deletes to the .json file.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout hcr-articleslist
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Articles from .json should display. User clicks "new article" to add an article. When added, new articles should display and be in permanent state in the .json server.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
